oxidation/client/src/views/WorkspacesPage.vue

```diff
@@ -117,127 +172,133 @@ async function openCreateWorkspaceModal(): Promise<void> {
-    .create({
-      header: 'Workspace name',
-      buttons: [
-        {
-          text: t('WorkspacesPage.workspaceContextMenu.share'),
-          icon: shareSocial,
-          handler: () :void => {
-            openWorkspaceShareModal();
-          }
-        },
-        {
-          text: t('WorkspacesPage.workspaceContextMenu.history'),
-          icon: time,
-          handler: ():void => {
-            console.log('History clicked');
-          }
-        },
-        {
-          text: t('WorkspacesPage.workspaceContextMenu.cancel'),
-          icon: close,
-          role: 'cancel',
-          handler: ():void => {
-            console.log('Cancel clicked');
-          }
-        }
-      ]
-    });
-  await actionSheet.present();
-  const { role, data } = await actionSheet.onDidDismiss();
-  console.log('onDidDismiss resolved with role and data: ', role, data);
+function onToggleView(value: boolean): void {
+  listView.value = value;
 }
+</script>
 
-async function openWorkspaceShareModal(): Promise<void> {
-  const modal = await modalController.create({
-    component: WorkspaceShareModal
-  });
-  modal.present();
+<style lang="scss" scoped>
 
-  const { data, role } = await modal.onWillDismiss();
+.workspaces-container {
+  margin: 2em;
+  background-color: white;
+}
 
-  if (role === 'confirm') {
-    console.log(data);
+.workspace-list-header {
+  color: var(--parsec-color-light-secondary-grey);
+  font-weight: 600;
+  padding-inline-start:0;
+
+  &__label {
+    padding: 0 1rem;
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .label-name {
+    width: 100%;
+    max-width: 20vw;
+    min-width: 11.25rem;
+    white-space: nowrap;
+    overflow: hidden;
+  }
+
+  .label-role {
+    min-width: 11.25rem;
+    max-width: 10vw;
+    flex-grow: 2;
+  }
+
+  .label-users {
+    min-width: 14.5rem;
+    flex-grow: 0;
+  }
+
+  .label-update {
+    min-width: 11.25rem;
+    flex-grow: 0;
+  }
+
+  .label-size {
+    min-width: 7.5rem;
+  }
+
+  .label-space {
+    min-width: 4rem;
+    flex-grow: 0;
+    margin-left: auto;
+    margin-right: 1rem;
   }
 }
-</script>
 
-<style lang="scss" scoped>
-.workspaces-grid-container {
-    display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(180px, 1fr));
+.workspaces-footer {
+  width: 100%;
+  left: 0;
+  position: fixed;
+  bottom: 0;
+  text-align: center;
+  font-weight: 600;
+  color: var(--parsec-color-light-secondary-text);
+  margin-bottom: 2em;
+}
+
+.workspaces-container-grid {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1.5em;
+  overflow-y: auto;
+
+}
+
+ion-item::part(native) {
+  --padding-start: 0px;
 }
 
 .workspace-toolbar {
-    --padding-start: 0px;
+  padding: 1em 2em;
+  height: 6em;
+  background-color: var(--parsec-color-light-secondary-background);
+  border-top: 1px solid var(--parsec-color-light-secondary-light);
 }
 
-.workspace-toolbar__change-view {
-    margin-left: auto;
+.rightSide {
```

Comment:
rightSide should be written right-side since it's CSS

Reply:
Done